.ci/install_kubernetes.sh

```diff
@@ -29,10 +30,12 @@ EOF"
 	chronic sudo -E apt update
 	chronic sudo -E apt install --allow-downgrades -y kubelet="$kubernetes_version" kubeadm="$kubernetes_version" kubectl="$kubernetes_version"
 elif [ "$ID" == "centos" ] || [ "$ID" == "fedora" ]; then
-	sudo bash -c "cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+	url="https://packages.cloud.google.com/yum/repos/kubernetes-el7-${ARCH}"
+        echo "Install ${url} for ${ARCH}"
```

Comment:
can you please fix indentation?

Reply:
@chavafg ,
Somehow I am not seeing these indentation at my local, also viewing the [file](https://github.com/kata-containers/tests/blob/641bbe8a8d8f5fc5a7e028b3b47b57db3c6655f1/.ci/install_kubernetes.sh#L34), looks good too. Not sure what missed here. thanks!

Reply:
@jing-wang4, hi, seems that you are using spaces and the other lines in the file are using tabs

Reply:
Thanks @chavafg !
I mistakenly got my fork deleted, so please review the duplicate/new [PR](https://github.com/kata-containers/tests/pull/3244) and close this one. Thanks!